srcpkgs/libtd/patches/cross-compiling.patch

```diff
@@ -0,0 +1,110 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Could this patch be replaced by a `vsed` command? Would make updates much simpler.